utils/anautilities.py

```diff
@@ -412,9 +402,23 @@ def getGemType(entry):
                 dict_DACvsADC_Funcs[dacName][ohKey][vfat].Write("func_VFAT{0}_DACvsADC_{1}".format(vfat,dacName))
                 dict_RawADCvsDAC_Graphs[dacName][ohKey][vfat].Write("g_VFAT{0}_RawADCvsDAC_{1}".format(vfat,dacName))
 
-                if vfat in dict_nonzeroVFATs[ohKey]:
-                    outputTxtFiles_dacVals[dacName][ohKey].write("{0}\t{1}\n".format(vfat,dict_dacVals[dacName][ohKey][vfat]))
+        for idx in range(len(dacNameArray)):
+            dacName = np.asscalar(dacNameArray[idx])
```

Comment:
do you need the loop index (i realize this code is probably written to match the code in the previous loop)?
can you not use a more pythonic iteration:
```python
for dacName in dacNameArray:
    dacName = np.asscalar(dacName) #if necessary
    blah
```